pyproject.toml

```diff
@@ -7,7 +7,6 @@ name = "pygmt"
 description = "A Python interface for the Generic Mapping Tools"
 readme = "README.rst"
 requires-python = ">=3.9"
-license = {text = "BSD License"}
```

Comment:
Is there a need to remove this line?

Reply:
See this comment https://github.com/GenericMappingTools/pygmt/pull/2957#discussion_r1443658544.

Reply:
Ah ok, I see the reason at https://github.com/GenericMappingTools/pygmt/pull/2957/files#r1443658544